docs/Grade.md

```diff
@@ -0,0 +1,71 @@
+
+```tsx
+import { Grade } from '@brightlayer-ui/react-native-components';
+
+<Grade label={'CS'} fontColor="#FFFFFF" backgroundColor="#198900" size={40} />;
+
+```
+
+### Grade API
+
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| label               | The text shown in Grade                    | string                                 | yes      |                        |
+| fontColor           | The color of the text label.               | string                                 | no       | theme.colors.onPrimary |
+| backgroundColor     | The background color of Grade              | string                                 | no       | theme.colors.primary   |
+| size                | The radius of Grade component              | number                                 | no       | 40                     |
+| theme           	  | Theme value overrides                      | `$DeepPartial<ReactNativePaper.Theme>` | no       |         				|
+
+
+## Grade Subcomponent
+
+The subcomponents gives you a set of preconfigured components that can directly be used 
+
+<img width="100%" style="max-width: 200px; display: block;" alt="grade subcomponent" src="./images/gradesSubcomponent.png">
+
+### Grade Subcomponents USAGE
+
+```tsx
+import Grade from '@brightlayer-ui/react-native-components';
+<View> 
+	<Grade.APlus /> {/* Renders A+ grade */} 
+	<Grade.A /> {/* Renders A grade */} 
+	<Grade.AMinus /> {/* Renders A- grade */}
+	<Grade.BPlus/> {/* Renders B+ grade */}
+	<Grade.B/> {/* Renders B grade */}
+	<Grade.BMinus/> {/* Renders B- grade */}
+	<Grade.CPlus/> {/* Renders C+ grade */}
+	<Grade.C/> {/* Renders C grade */}
+	<Grade.CMinus/> {/* Renders C- grade */}
+	<Grade.DPlus/> {/* Renders D+ grade */}
+	<Grade.D/> {/* Renders D grade */}
+	<Grade.DMinus/> {/* Renders D- grade */}
+	<Grade.F/> {/* Renders F grade */}
+</View>
+```
+### Grade Subcomponents API
+
+All Grade Subcomponents have the following Props
+
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| size                | The radius of Grade component              | number                                 | No       | 40                     |
+
+### Styles
+
+You can override the internal style used by Brightlayer UI by passing a `style` prop. The Grade supports the following keys:
+
+```tsx
+<Grade style={{ marginBottom: 10 }} />
+```
+
+| Name | Description                                 |
+| ---- | ------------------------------------------- |
+| root | Style override for the root View element |
```

Comment:
This part doesn't make sense since this component doesn't have a `styles` prop with multiple keys like the other ones (I don't think it's necessary). We can skip the part about keys and just leave the first bit.
```suggestion
### Styles

You can override the internal style used by Brightlayer UI by passing a `style` prop. 
```